working-groups.md

```diff
@@ -0,0 +1,168 @@
+| <img width="30px" src="https://github.com/danielhelfand.png"> | Daniel Helfand    | VMWare  | [danielhelfand](https://github.com/danielhelfand) |
+
+## Documentation
+
+This is the documentation working group, related to
+[`tektoncd/website`](https://github.com/tektoncd/website) and all
+repository that need to expose documetnation.
+
+| Artifact                   | Link                                                                                                                                                                                                                                         |
+| -------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------                                                                                |
+| Forum                      | [tekton-dev@](https://groups.google.com/forum/#!forum/tekton-dev)                                                                                                                                                                            |
+| Community Meeting VC       | [https://meet.google.com/hyc-nunv-hts](https://meet.google.com/hyc-nunv-hts)                                                                                                                                                                 |
+| Community Meeting Calendar | Wednesdays every 2 weeks, 01:00p-01:30p EST <br>[Calendar](https://calendar.google.com/event?action=TEMPLATE&tmeid=MW9oZzNpbmkxbHU5ZGtlN2hnMzVsYjNpcTJfMjAxOTEyMDRUMTgwMDAwWiB2ZGVtZWVzdEByZWRoYXQuY29t&tmsrc=vdemeest%40redhat.com&scp=ALL) |
+| Meeting Notes              | [Notes](https://www.google.com/url?q=https://docs.google.com/document/d/1nnJe4rWBFZORW9ScV9o-yk6BmvX419DDKKwIlORRhsY/edit%23&sa=D&ust=1575716141397000&usg=AOvVaw0IMqtfI1KeX8a6zq2ODl9n)                                                     |
+| Document Folder            | [Folder](https://drive.google.com/drive/folders/1dpGatrFh_Wykrs_tUkVK8hLNF1VaGHZd)                                                                                                                                                           |
+| Slack Channel              | [#docs](https://tektoncd.slack.com/messages/docs)                                                                                                                                                                                            |
+
+
+| &nbsp;                                                     | Facilitators      | Company | Profile                                     |
+| --------------------------------------------------------   | ----------        | ------- | ---------------------------------------     |
+| <img width="30px" src="https://github.com/vdemeester.png"> | Vincent Demeester | Red Hat | [vdemeester](https://github.com/vdemeester) |
```

Comment:
i think @spomorski and/or @kimsterv might be interested in joining as facilitators also

Reply:
Right, I wasn't sure who to add :angel: forgot to comment on that :angel:

Reply:
now it looks like you are on there twice ;)

Reply:
:pray-panda:

Reply:
:man_facepalming: never edit a file too late :joy_cat:

Reply:
XD